mshadow/tensor_cpu-inl.h

```diff
@@ -331,7 +331,7 @@ inline void SoftmaxGrad(Tensor<cpu, 3, DType> dst,
   for (openmp_index_t n = 0; n < dst.size(2); ++n) {
```

Comment:
I've seen things like the dist.size(2) in the for comparison loop show up as hotspots in vtune a many times, even though people generally think that the compiler inlines them to the same thing as a scalar variable.